src/main/charts/jira/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: jira
 description: A chart for installing Jira Data Center on Kubernetes
 type: application
 version: 1.7.1
-appVersion: 8.20.14
```

Comment:
This is a new Major LTS, do we need more rigorous testing than our current automation?